css/index.css

```diff
@@ -1,3 +1,293 @@
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+@media screen and (max-width: 800px){
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: center;
+        align-items: center;
+    }
+    
+    header .logo {
+        display: flex;
+        width: 25%;
+    }
+
+    .cta {
+        display: flex;
+        flex-wrap: nowrap;
+        flex-direction: column;
+    }
+    
+    .cta img {
+        display: none;
+    }
+}
+
+@media screen and (max-width: 500px){
+    body {
+        display: flex;
+        justify-content: center;
+    }
+    
+    header .logo {
+        margin: 2% 0%;
+    }
+
+    header nav {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    header nav a {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-size: 20px;
+        border-bottom: .25px solid black;
+        width: 499px;
+        margin: 0px;
+        padding: 15px 0px;
+    }
+
+    header nav a:first-child {
+        border-top: 1px solid black;
+      }
+
+      header nav a:last-child {
+        border-bottom: 1px solid black;
+      }
+    
+
+    .cta .cta-text {
+        margin: 10% 0;
+    }
+
+    .main-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        border-top: 2px solid black;
+        margin: 0 20%;
+    }
+
+    .main-content p {
+        padding: 5% 0 10% 0;
+    }
+
+    .main-content .middle-img {
+        margin-top: 45px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 500px;
+    }
```

Comment:
<h3>If you want your image to be contained inside the 500px, try making it smaller than that. Remember to account for width and margin! If the width of the image and margin exceeds 500px you will get that horizontal scroll bar that you wanna avoid.<h3>